app/src/Doctrine/Website/DoctrineSculpinBundle/Directive/ToctreeDirective.php

```diff
@@ -18,34 +19,26 @@ public function process(Parser $parser, $node, $variable, $data, array $options)
         $kernel = $parser->getKernel();
         $files = array();
 
-        // This sucks. I don't even think it works 100% right.
```

Comment:
It still sucks but it is better and I am more confident now in how it should work and how it works now. I am going to keep on working on this as I move through the documentation for all the projects.

Reply:
I should also add that I'm going to move ToctreeDirective to doctrine/rst fork and move the associated tests there.